pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1334,13 +1334,27 @@ func (c *PodAffinityChecker) satisfiesExistingPodsAntiAffinity(pod *v1.Pod, meta
 //  topology of all the "terms" for the given "pod".
 func (c *PodAffinityChecker) nodeMatchesAllTopologyTerms(pod *v1.Pod, topologyPairs *topologyPairsMaps, nodeInfo *schedulercache.NodeInfo, terms []v1.PodAffinityTerm) bool {
```

Comment:
@bsalamat @Huang-Wei I think ``pod *v1.Pod`` can be removed from the parameters as it's not used anymore.

Reply:
@ahmad-diaa Thanks for the catch.